i18n/en.json

```
@@ -0,0 +1,16 @@
+{
+    "bot_username": "Bot Username",
+    "dialogflow_project_id": "Dialogflow Project Id",
```

```suggestion
    "dialogflow_project_id": "Project Id",
```

Repeat the same for the next ones.

---

Okay made this change